Allow alternative sort directions, refs #12000

jraddaoui opened the pull request:
- Include another generic picker with sort direction options to the
  default sort picker and rename it to 'sortPickers'.
- Layout and style improvements.
- Handle sortDir param in actions.
- Rename sort option labels.

Pages that will have sort direction options after this changes:

- Actor browse
- Function browse
- IO browse
- Repository browse
- Rights holders browse
- Taxonomy index
- Term index
- Clipboard
- Accession browse
- Donor browse

sbreker reviewed (approved):
Looks good @jraddaoui!

jraddaoui committed:
Allow alternative sort directions, refs #12000

- Include another generic picker with sort direction options to the
  default sort picker and rename it to 'sortPickers'.
- Layout and style improvements.
- Handle sortDir param in actions.
- Rename sort option labels.

Pages that will have sort direction options after this changes:

- Actor browse
- Function browse
- IO browse
- Repository browse
- Rights holders browse
- Taxonomy index
- Term index
- Clipboard
- Accession browse
- Donor browse

Squashed commit of the following:

commit 3f3b8b04a8843333a833bb265cd7dc90e7f348d7
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Tue Mar 6 19:43:36 2018 +0100

    Change sort option labels

    Use 'Date modified' instead of 'Last updated' and 'Title' or 'Name' based
    on the entity instead of 'Alphabetic'. Modify settings page accordingly

commit 7970190b9f6dc3efce1b1ddbaef1efb214d55c56
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Tue Mar 6 15:17:48 2018 +0100

    Handle sortDir param in actions

commit 4a5ea7fe489e9e453318eb0150de28fd4dca5ef6
Author: José Raddaoui Marín <raddaouimarin@gmail.com>
Date:   Tue Feb 27 21:08:31 2018 +0100

    Add sort direction picker alongside sort picker

    - Include another generic picker with sort direction options to the
    default
      sort picker and rename it to 'sortPickers'.
    - Layout and style improvements.
